docs/esterni/manuale_installazione/manuale_installazione.tex

```diff
@@ -0,0 +1,285 @@
+\\È necessario clonare la repository GitHub al seguente indirizzo: \url{https://github.com/iota97/WinningSoftwareSolution}.
+\section{Server}
+\subsection{Requisiti}
+\subsubsection{Node Js}
+Installare \href{https://nodejs.org/en/download/}{Node Js} se non è già stato installato precedentemente. Una volta effettuata correttamente l'installazione, è necessario eseguire il seguente comando direttamente nella cartella "Server" per l'installazione delle dipendenze:
+\begin{lstlisting}[language=bash]
+  $ npm install
+\end{lstlisting}
+\subsubsection{Provider}
+\label{sec:provider}
+Per ottenere il provider per la testnet di Polygon Mumbai è necessario registrarsi su \href{https://www.moralis.io}{moralis.io}. Una volta effettuata la registrazione:
+\begin{enumerate}
+  \item selezionare la sezione "SpeedyNodes" dal menù;
+  \item selezionare l'endpoint per Polygon Network;
+  \item selezionare il protocollo WS;
+  \item copiare l'url dell'endpoint per la testnet Mumbai (Fig. 1).
+\end{enumerate}
+Nell'url copiato è possibile visualizzare le api key, si tratta infatti del codice evidenziato in grassetto nel seguente esempio:\\\\
+wss://speedy-nodes-nyc.moralis.io/\textbf{a8d734a415dd368a3498db63}/polygon/mainnet/ws\\\\
+È necessario salvare l'url e le api key per i passi successivi.
+\FloatBarrier
+\begin{figure}[!h]
+\centering
+\includegraphics[width=0.6\linewidth]{img/moralis.png}
+\caption{Provider su moralis.io.}
+\end{figure}
+\FloatBarrier
+\subsubsection{MariaDB}
+\label{sec:db}
+Installare il database \href{https://mariadb.org/download/}{MariaDB} se non è gia stato installato precedentemente. Una volta eseguita l'installazione del database, collegarsi con il comando:
+\begin{lstlisting}[language=bash]
+  $ sudo mysql -u <user> -p <password>
+\end{lstlisting}
+e sostituire:
+\begin{itemize}
+  \item \verb|<user>|: con l'user con il quale si desidera creare il database;
+  \item \verb|<password>|: con la password dell'user selezionato.
+\end{itemize}
+Effettuato il collegamento eseguire la seguente query per la creazione del database:
+\begin{verbatim}
+CREATE DATABASE OnlineStore;
+\end{verbatim}
+Successivamente creare le tabelle con le seguenti query:
+\begin{verbatim}
+USE OnlineStore;
+DROP TABLE PaymentEntries;
+DROP TABLE SettledPayments;
+DROP TABLE LastBlockSynced;
+CREATE TABLE PaymentEntries (id bigint, ecommerce varchar(255) not null,
+price bigint not null, primary key(id));
+CREATE TABLE SettledPayments (id bigint, item_id bigint not null,
+buyer varchar(255) not null, status int not null, created bigint not null,
+confirmed bigint, primary key(id));
+CREATE TABLE LastBlockSynced (id int(1), value bigint not null, primary key(id));
+INSERT INTO LastBlockSynced (id, value) VALUES (0, 0);
+\end{verbatim}
+\subsubsection{Metamask}
+Per poter utilizzare la webbapp è necessario metamask. Visitare \href{https://www.metamask.io}{metamask.io} per maggiori informazioni.\\\\
```

Comment:
Allora, come manuale di installazione mi sembra un po' scarno. Come si crea un wallet? Scriviamo che si può usare metamask ma in realtà un wallet può già essere in possesso dell'azienda o essere creato con altri software e anche offline. Scriviamo che è necessario arrivare ad avere la mnemonica di un wallet, poi a come ci si arriva si fa l'esempio (scrivendo esattamente come fare) con metamask e dicendo che si può fare in altro modo anche imo